github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -136,6 +136,13 @@ plugins:
     - owners-label
     - trigger
 
+  kubevirt/ipam-extensions:
+    plugins:
+    - approve
+    - lgtm
+    - owners-label
+    - trigger
```

Comment:
We don't need trigger, since we don't have prow lanes.

Reply:
Removed

imho it doesnt hurt it was there (anyhow removed)